app/views/super_admin/libraries/_form.html.erb

```diff
@@ -1,8 +0,0 @@
-<%= form_with(model: library, method: :patch, url: super_admin_library_url(library), builder: SpectreFormBuilder, data: {controller: "form"}) do |form| %>
```

Comment:
Since the forms are both the same, can we put them back here?

Reply:
The method and URL entires change on the different forms though, right?

Reply:
They didn't before and seemed to work... I wonder why we needed any more of a change than simply adding the image field.